test/integration/generated/headerprefix/src/headerPrefixClient.ts

```diff
@@ -39,6 +40,29 @@ export class HeaderPrefixClient extends coreClient.ServiceClient {
       baseUri: options.endpoint ?? options.baseUri ?? "{$host}"
     };
     super(optionsWithDefaults);
+
+    if (options?.pipeline && options.pipeline.getOrderedPolicies().length > 0) {
+      const pipelinePolicies: coreRestPipeline.PipelinePolicy[] = options.pipeline.getOrderedPolicies();
+      const bearerTokenAuthenticationPolicyFound = pipelinePolicies.find(
```

Comment:
nit: if you just want to know if the policy is present you can use pipelinePolicies.some which gives you a boolean